packages/material-ui/src/ImageList/ImageList.d.ts

```diff
@@ -19,17 +16,35 @@ export interface ImageListTypeMap<P = {}, D extends React.ElementType = 'ul'> {
     classes?: {
       /** Styles applied to the root element. */
       root?: string;
+      /** Styles applied to the root element if `variant="masonry"`. */
+      masonry?: string;
+      /** Styles applied to the root element if `variant="quilted"`. */
+      quilted?: string;
+      /** Styles applied to the root element if `variant="standard"`. */
+      standard?: string;
+      /** Styles applied to the root element if `variant="woven"`. */
+      woven?: string;
     };
     /**
      * Number of columns.
      * @default 2
      */
     cols?: number;
     /**
-     * Number of px for the spacing between tiles.
+     * The height of one row in px.
+     * @default 'auto'
+     */
+    rowHeight?: number | 'auto';
+    /**
+     * The spacing between items in px.
      * @default 4
      */
     spacing?: number;
```

Comment:
Should we rename it to match the name of CSS feature (also coming to flexbox)?

```suggestion
    gap?: number;
```